sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -324,7 +324,8 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
       DescribeTableCommand(
         visitTableIdentifier(ctx.tableIdentifier),
         partitionSpec,
-        ctx.EXTENDED != null || ctx.FORMATTED != null)
+        ctx.EXTENDED != null,
+        ctx.FORMATTED != null)
```

Comment:
Oh, I see. Then, you want remove the header completely for all case. Did I understand correctly?